configs/configs.py

```diff
@@ -0,0 +1,11 @@
+
+# Discord Bot Settings
+DISCORD_TOKEN = ""
+
+# Event listeners
+ENABLE_SOCIAL_CREDIT = False
```

Comment:
Since the goal is to make this bot usable by anyone by downloading the code, maybe someone wont want social credit continuously running. And maybe if they do they run their own emojis.

There is a `is_enabled` function in `CommandBase` that can be inherited and changed to add or remove functionality